VogtPayroll5/Employee.cs

```diff
@@ -27,13 +27,25 @@ public decimal CalculateGrossPay(int hours, decimal rate)
             }
         }
 
+        public decimal CalculateOvertimePay(int hours, decimal rate)
+        {
+            int baseHours = 40;
+
+            if (hours >= 40)
+                hours = hours - baseHours;
+            else
+                return 0;
+
+            return hours * rate;
+        }
+
         public void DisplayPayStatement(int empID, string empName, int hoursWorked, decimal hourlyPayRate, decimal grossPay)
         {
             Console.WriteLine($"Employee ID: {empID}");
             Console.WriteLine($"Employee name: {empName}");
             Console.WriteLine($"Employee hours worked: {hoursWorked}");
-            Console.WriteLine($"Employee hourly payrate: {hourlyPayRate}");
-            Console.WriteLine($"Employee gross pay: {CalculateGrossPay(hoursWorked, hourlyPayRate)}");
+            Console.WriteLine($"Employee hourly payrate: {hourlyPayRate:C2}");
+            Console.WriteLine($"Employee gross pay: {CalculateGrossPay(hoursWorked, hourlyPayRate) + CalculateOvertimePay(hoursWorked, hourlyPayRate):C2}");
```

Comment:
The fact that the gross pay is CalculateGrossPay() + CalculateOvertimePay() is a code smell. If the gross pay includes the overtime, then perform the overtime calculation within the CalculateGrossPay() function.

If you really break things down, it is not the DisplayPayStatement function's responsbility to know how to calculate the gross pay -- thats the responsbility of `CalculateGrossPay()`